seatunnel-e2e/seatunnel-connector-v2-e2e/pom.xml

```diff
@@ -28,6 +28,7 @@
         <module>connector-jdbc-e2e</module>
         <module>connector-redis-e2e</module>
         <module>connector-clickhouse-e2e</module>
+        <module>connector-tikv-e2e</module>
```

Comment:
Module `connector-tikv-e2e` not committed?

Reply:
> Module `connector-tikv-e2e` not committed?

revert file add module